python/pyspark/sql/tests/connect/test_parity_dataframe.py

```diff
@@ -120,8 +120,7 @@ def test_same_semantics_error(self):
     def test_sample(self):
         super().test_sample()
 
-    # TODO(SPARK-41875): throw proper errors in Dataset.to()
```

Comment:
So, it sounds like now we throws proper errors, but the test still disabled ??

If so, maybe we should say like:
```python
# TODO(SPARK-XXXXX): Implement Dataset `to`
@unittest.skip("Fails in Spark Connect, should enable.")
```

??

Reply:
Yeah we should fix the test like https://github.com/apache/spark/pull/39450#discussion_r1064065514

Reply:
I meant that exception. We should ideally fix the test to catch both `AnalysisException` and `SparkConnectAnalysisException`. So let's file a new JIRA, and fix the comment here